crates/rnote-cli/src/export.rs

```diff
@@ -0,0 +1,673 @@
+use std::{
+    path::{Path, PathBuf},
+    time::Duration,
+};
+
+use anyhow::Context;
+use clap::{Args, Subcommand, ValueEnum};
+use parry2d_f64::{bounding_volume::Aabb, na::Vector2};
+use rnote_compose::SplitOrder;
+use rnote_engine::{
+    engine::{
+        export::{
+            DocExportFormat, DocExportPrefs, DocPagesExportFormat, DocPagesExportPrefs,
+            SelectionExportFormat, SelectionExportPrefs,
+        },
+        EngineSnapshot,
+    },
+    selectioncollision::SelectionCollision,
+    RnoteEngine,
+};
+use smol::{
+    fs::File,
+    io::{AsyncReadExt, AsyncWriteExt},
+};
+
+use crate::cli::OnConflict;
+use crate::validators;
+
+#[derive(Subcommand, Debug)]
+pub(crate) enum ExportCommands {
+    /// Export the entire document.{n}
+    /// When using --output-file, only one input file can be given.{n}
+    /// The export format is recognized from the file extension of the output file.{n}
+    /// When using --output-format, the file name and path of the rnote file is used with the extension changed.{n}
+    /// --output-file and --output-format are mutually exclusive but one of them is required.{n}
+    /// Currently `.svg`, `.xopp` and `.pdf` are supported.{n}
```

Comment:
Remove this line, the possible options are listed and this feels out of context here.

Reply:
I changed it to say that --output-file currently supports ...

Or do you think the possible file extensions are clear from the possible values for output-format?

Reply:
yes, it is enough when they are listed at the help text of the `--output-format` argument